tests/forms_tests/field_tests/test_uuidfield.py

```diff
@@ -18,7 +18,8 @@ def test_uuidfield_2(self):
 
     def test_uuidfield_3(self):
         field = UUIDField()
-        with self.assertRaises(ValidationError) as cm:
+        msg = 'Enter a valid UUID.'
+        with self.assertRaisesMessage(ValidationError, msg) as cm:
             field.clean('550e8400')
         self.assertEqual(cm.exception.messages[0], 'Enter a valid UUID.')
```

Comment:
I forget if there was a reason to prefer the current version but yes, we don't need the check twice.